Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -165,6 +165,27 @@ class PushNotificationIOS {
   }
 
   /**
+   * Remove all delivered notifications from Notification Center
+   */
+  static removeAllDeliveredNotifications() {
+    RCTPushNotificationManager.removeAllDeliveredNotifications();
+  }
+
+  /**
+   * Provides you with a list of the app’s notifications that are still displayed in Notification Center
+   */
+  static getDeliveredNotifications(callback: Function) {
```

Comment:
I didn't mean adding an invariant, I meant using Flow's function types like this: https://flowtype.org/docs/functions.html